packages/inertia-svelte/package.json

```diff
@@ -19,13 +19,27 @@
   "keywords": [
     "svelte"
   ],
-  "main": "src/index.js",
+  "module": "dist/index.mjs",
+  "main": "dist/index.js",
   "devDependencies": {
-    "@inertiajs/inertia": "^0.8.0",
-    "eslint": "^7.0.0"
+    "@inertiajs/inertia": "^0.8.5",
+    "@rollup/plugin-commonjs": "^17.1.0",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-node-resolve": "^11.2.0",
+    "eslint": "^7.21.0",
+    "rollup": "^2.40.0",
+    "rollup-plugin-peer-deps-external": "^2.2.4",
+    "rollup-plugin-svelte": "^7.1.0",
+    "rollup-plugin-terser": "^7.0.2",
+    "svelte": "^3.35.0"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "watch": "rollup -c -w",
+    "prepublishOnly": "npm run build"
   },
   "peerDependencies": {
-    "@inertiajs/inertia": "^0.8.0",
-    "svelte": "^3.20.0"
+    "@inertiajs/inertia": "0.8.5",
```

Comment:
Do we need `@inertiajs/inertia`  as both `devDependencies` and a `peerDependencies`? Shouldn’t it be one or the other?

Reply:
Nope. Good catch.